src/pages/CandlestickPage.vue

```diff
@@ -0,0 +1,127 @@
+<script lang="ts">
+import {
+  CandlestickData,
+  ChartOptions,
+  createChart,
+  IChartApi,
+  ISeriesApi,
+  Time
+} from 'lightweight-charts';
+import { defineComponent } from 'vue';
+
+const data: CandlestickData[] = [
+  { open: 10, high: 10.63, low: 9.49, close: 9.55, time: 1642427876 as Time },
+  { open: 9.55, high: 10.3, low: 9.42, close: 9.94, time: 1642514276 as Time },
+  { open: 9.94, high: 10.17, low: 9.92, close: 9.78, time: 1642600676 as Time },
+  { open: 9.78, high: 10.59, low: 9.18, close: 9.51, time: 1642687076 as Time },
+  { open: 9.51, high: 10.46, low: 9.1, close: 10.17, time: 1642773476 as Time },
+  {
+    open: 10.17,
+    high: 10.96,
+    low: 10.16,
+    close: 10.47,
+    time: 1642859876 as Time
+  },
```

Comment:
Lets write all the entries like this

```suggestion
  { open: 10.17, high: 10.96, low: 10.16, close: 10.47, time: 1642859876 as Time },
```